src/test/func/memcpy/func-memcpy.cc

```diff
@@ -82,7 +82,7 @@ void check_size(size_t size)
     {
       dst[i] = 0;
     }
-    my_memcpy(dst, src, sz);
+    snmalloc::memcpy(dst, src, sz);
```

Comment:
I don't really like this because it means that we need to do extra work to hide the symbol from the .so.  We can't use `snmalloc::memcpy<false>` in any other compilation unit.